lib/colvars/lepton/src/MSVC_erfc.h

```diff
@@ -8,9 +8,13 @@
  * (VC11 has _MSC_VER=1700).
  */
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
This is okay with me, of course. However, I'll submit a PR to https://github.com/openmm/openmm so that it is consistent with the upstream version of this file.

Reply:
Thanks @giacomofiorin. I would have done it myself after we get the upcoming patch release out of the door. This one is going to have quite a few significant changes, so I am a bit nervous to make sure we have all bases covered.

Reply:
@akohlmey Sounds good. 

In a related question, although MingW doesn't use `dllexport` or `dllimport` (does it?), there is a define flag to be added to have Lepton's Windows header to behave correctly with MSVC. I'll add it as a separate commit on my LAMMPS fork, and you can cherry-pick it when/where appropriate.

Reply:
don't know whether this is needed. i recently wanted to know what happened when i set `BUILD_SHARED_LIBS` when cross-compiling with CMake and it worked like a charm(m?).
Could run the executable (after copying the .dll where the .exe was) and even could import the shared lib into python and run some simulation with the python wrapper class (after teaching `lammps.py` that .dll is a possible shared lib suffix).

Reply:
I submitted PR #1997. Best to deal with it before someone experiments with MSVC, and it brings y'all one step closer closer to #2000...
If it did work like a charmm (i.e. with two "m"s) then it was probably working correctly for the wrong reason. I'd keep that in mind next time you touch that house of cards ;-)

Reply:
Well, I've managed to call LAMMPS from python, which is the easy part, now I have to manage calling python from LAMMPS. I have seen, that there are now python3 packages for the mingw cross-compiler toolchain, so the compiling should be possible, the running may be more tricky. there are lots of worms in this can...